paasta_tools/long_running_service_tools.py

```diff
@@ -206,6 +206,26 @@ def get_instances(self) -> int:
             log.debug("Autoscaling not enabled, returning %d instances" % instances)
             return instances
 
+    def get_instances_without_limit(self) -> int:
```

Comment:
How about just making this a param on `get_instances`. Something like `get_instances(self, with_limit=True)` so that it defaults to the current behaviour.

Reply:
Sounds good. Making the change.